modules/setting/setting.go

```diff
@@ -1256,6 +1266,12 @@ var Service struct {
 	OpenIDBlacklist    []*regexp.Regexp
 }
 
+var visibilityModes = map[string]int{
+	"public":  1,
```

Comment:
reuse constants also here.. Also are they are correct? (in code in model it is used as 0, 1, 2 - might be it is incorrect in selects...)

Reply:
Can't import constants into settings because cyclical (file with constants already imports settings). They are correct, the reason why 0 is used in select is because users (who don't have visibility) are set to 0 and org/user search uses the same code because they are essentially the same. So I have to make sure that if a code needs to search for users then users are included.

Reply:
Maybe we should move all structs to `models/structs` as a sub package.

Reply:
done.